common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -110,15 +123,16 @@ def test_page_view_event(self):
         And I visit my account settings page
         Then a page view analytics event should be recorded
         """
-        self.verify_events_of_type(
-            self.username,
-            u"edx.user.settings.viewed",
-            [{
-                u"user_id": long(self.user_id),
-                u"page": u"account",
-                u"visibility": None,
-            }]
-        )
+
+        event_filter = {
+            'event_type':'edx.user.settings.viewed',
+            'event': {
+                'user_id': self.user_id,
+                'page': 'account',
+                'visibility': None
+            }
+        }
+        self.wait_for_events(event_filter=event_filter, number_of_matches=1)
```

Comment:
@benpatterson I think I understand, I'll take a pass at this breakdown and tag you once I upload it.